webpack.config.js

```diff
@@ -1,71 +1,52 @@
 const path = require('path');
+const fs = require('fs');
 
-module.exports = {
-    entry: './src/index.ts',
+const currentWorkingDirectory = __dirname;
 
-    /*
-        Externals below defines those peer dependencies that should not be bundled in global output
-        file and instead be loaded by the consumer prior to loading the distributable that webpack
-        generates.
+const configPromise = new Promise(function (resolve, reject) {
 
-        NOTE: Any externals that are commented out below are being included in the webpack
```

Comment:
Question: Why did you end up breaking this out into separate files for bundled/externals? Personally, it seemed nice to have it all in one spot with comments for those items that are bundled (and could even have been consolidated down to a single comment `// bundled: queryString` for example.

Reply:
Originally what was proposed was to read a JS script file `webpack.config.js` which is clearly less than ideal.  Instead, I thought that this simplified the process by having it read in strict JSON file formatting.  Now it's easy enough to describe (see `CONTRIBUTING.md`) and understand that new peer dependencies need to be identified in one of two different JSON files.  I can certainly revert this, but i really felt that reading JS gets ugly and is more fragile.

Reply:
Fair enough, I'm good to leave it the way you have it, as this grows it will certainly keep it a bit cleaner.